yt/visualization/fits_image.py

```diff
@@ -906,17 +910,20 @@ def construct_image(
             frb = data_source.to_frb(width[0], (nx, ny), height=width[1])
         else:
             frb = data_source.to_frb(width[0], (nx, ny), center=center, height=width[1])
-    elif isinstance(data_source, ParticleAxisAlignedDummyDataSource):
-        axes = axis_wcs[axis]
-        bounds = (
-            center[axes[0]] - width[0] / 2,
-            center[axes[0]] + width[0] / 2,
-            center[axes[1]] - width[1] / 2,
-            center[axes[1]] + width[1] / 2,
-        )
-        frb = ParticleImageBuffer(
-            data_source, bounds, (nx, ny), periodic=all(ds.periodicity)
-        )
+    elif isinstance(data_source, ParticleDummyDataSource):
+        if hasattr(data_source, "normal_vector"):
+            bounds = (-width[0] / 2, width[0] / 2, -width[1] / 2, width[1] / 2)
+            periodic = False
```

Comment:
I don't think I get this. Wouldn't an off-axis projection be periodic if axis-boundaries are *all* periodic ?

Reply:
if I recall correctly we do not support periodicity for any off-axis projection plots. Isn't that correct, @matthewturk?

Reply:
This is my recollection.